src/misc.jl

```diff
@@ -201,3 +201,42 @@ end
     root --> :top
     GraphPlot((source, destiny))
 end
+
+
+@userplot AndrewsPlot
+
+"""
+https://en.wikipedia.org/wiki/Andrews_plot
+
+Example
+```julia
+using RDatasets, PlotRecipes
+iris   = dataset("datasets", "iris")
+y      = Array(iris[:,1:4])
+groups = Array(iris[:,5])
+andrewsplot(y, groups)
+```
+"""
+@recipe function f(h::AndrewsPlot)
+    y = h.args[1]
+    seriestype --> :path
+    title --> "Andrew's plot"
+    label := ""
+    rows,cols = size(y)
+    serieslength = 100
+    ys    = zeros(serieslength,rows)
+    t     = linspace(-π,π,serieslength)
+    for j in 1:rows, ti = eachindex(t)
+        ys[ti,j] = y[j,1]/sqrt(2) + sum(y[j,i].*sin.((i÷2).*t[ti]) for i = 2:cols)
+    end
+    if length(h.args) >= 2
```

Comment:
I think I solved it with the `primary` attribute like
```julia
            @series begin
                groupinds = g .== kind
                prim    = falses(sum(groupinds))
                prim[1] = true
                primary --> prim'
                t,ys[:,groupinds]
            end
```

One problem remains, the `group` keyword can not be used. If I try to use it my recipe function is called several times, probably a conflict with the standard keyword `group`. I will upload a working solution with another keword so you can se how it looks atm